src/api/get.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+module.exports = (send) => {
+  return function get (path, archive, compress, compressionLevel, cb) {
+    if (archive === true && typeof compress === 'function') {
```

Comment:
Agreed, please make `archive`, `compress` and `compressionLevel` into an options object.